pkg/kritis/admission/admission.go

```diff
@@ -243,6 +247,15 @@ func checkBreakglass(pod *v1.Pod) bool {
 	return ok
 }
 
+func checkBreakglassDeployment(deployment *appsv1.Deployment) bool {
```

Comment:
Not sure if this would work, but could we have something like

```golang
func checkBreakglass(meta *ObjectMeta){}
```

so that we don't have to separate the checks for pods and deployments? Just wondering if that could work so we can use it for future types we add as well.

Reply:
wasn't able to get this to work, keeping for now